src/styles/scss/tools/placeholder.scss

```diff
@@ -1,4 +1,3 @@
-// stylelint-disable plugin/stylelint-bem-namics
 %text-truncate {
   white-space: nowrap;
   overflow: hidden;
```

Comment:
Can we remove this entirely, and just use the mixin instead? 

I see you added it for with param 1 line for .card__tags, and now i think it's only used one place in event-list. 

Up to you.